app-tasks/Dockerfile

```diff
@@ -17,16 +17,13 @@ RUN set -ex \
     && buildDeps=' \
        python-dev \
     ' \
-    && sparkDeps=' \
```

Comment:
Because the JDK is installed for Spark, I think that all of the Spark dependencies can be removed as well. Essentially reverting https://github.com/azavea/raster-foundry/commit/22e0a76c6e129bd8a6965a53571a8c9379a07e54.